packages/simple-exchange-frontend/yarn.lock

```diff
@@ -0,0 +1,11960 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think this file should not exist? We should only have a single yarn.lock at the top level.